test/test_models.py

```diff
@@ -40,22 +52,35 @@ def get_available_video_models():
 # they are not yet supported in JIT.
 script_test_models = [
     "deeplabv3_resnet101",
-    "mobilenet_v2",
```

Comment:
Those changes seems not related to this PR?

Reply:
The classification models are all tested for scriptability.